pkg/service/backtest.go

```diff
@@ -290,3 +290,15 @@ func (s *BacktestService) BatchInsert(kline []types.KLine) error {
 	_, err := s.DB.NamedExec(sql, kline)
 	return err
 }
+
+func (s *BacktestService) DeleteDuplicatedKLine(k types.KLine) error {
```

Comment:
make it as a private method? since it's used internally?

Reply:
it's used in cmd/backtest . so if we made it private it won't be able to access there.